runtime/util/argcount.c

```diff
@@ -20,19 +20,38 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
-
-
 #include "j9.h"
 
-/* include '[' for arrays (character after 'Z') */
+/* These represent argcounts for java field descriptors, characters without descriptions are currently unused */
 const U_8 argCountCharConversion[] = {
-0,	1,	1,	2,
-0,	1,	0,	0,
-1,	2,	0,	1,
-0,	0,	0,	0,
-0,	0,	1,	0, 
-0,	0,	0,	0,
-0,	1,	1,	0}; 
+    0, /* A */
+    1, /* B -> byte */
+    1, /* C -> char */
+    2, /* D -> double */
+    0, /* E */
+    1, /* F -> float */
+    0, /* G */
+    0, /* H */
+    1, /* I -> int */
+    2, /* J -> long */
+    0, /* K */
+    1, /* L -> nullable named class or interface type */
+    0, /* M */
+    0, /* N */
+    0, /* O */
+    0, /* P */
+    1, /* Q -> null-free named inline class type (also known as value type) */
+    0, /* R */
+    1, /* S -> short */
+    0, /* T */
+    0, /* U */
+    0, /* V -> void */
+    0, /* W */
+    0, /* X */
+    0, /* Y */
+    1, /* Z -> boolean */
+    1  /* [ -> 1-D array reference */
+}; 
```

Comment:
The array size here has changed from 28 to 27. The stripped character is /, so this should not matter, but I'd like it verified that the change was intentional.

Reply:
Yes, I intentionally removed it because it was a character that isn't currently used to represent a java field descriptor.